Justfile

```diff
@@ -45,18 +45,46 @@ run-ci-tests:
 
 path_prefix := "api/upgrades_info/"
 
+display-graph:
+	jq -cM . | {{invocation_directory()}}/hack/graph.sh | dot -Tsvg > graph.svg; xdg-open graph.svg
```

Comment:
We need to add the graph.svg to `.gitignore`.

Reply:
For some reason this is not working in my tmux session. But thats not a blocker for the PR.

Reply:
Please paste the command and its output here. It's supposed to work for you as well.

Reply:
I guess I am using `$ just display-graph` in a wrong way. Can you add an example in the dev docs?

Reply:
Running `$ just get-and-display-graph` works fine. But as expected `$ just display-graph` would hang as it need `json` input. If this is right, we should make `$ just display-graph` a private recipe.

Reply:
@steveeJ  mentioned that the right usage is `just get-graph-pe-production "stable-4.2" "amd64" | just display-graph`. As discussed we should add  docs for ` display-graph` in the Justfile for `just --list` to show. I will also suggest that we add `just --list` to the dev docs.

Reply:
I'm not yet ready to document all of this. I would like to, but I'm not convinced this is a stable interface. Right now we're observing a divergence, or at best a duplication, between the Justfile and the scripts in _dist/_ and _hack/_. I would like to unify this as much as possible soon along with updating the documentation.

Reply:
I think the divergence exists because we have not accepted the Justfile yet as the default way for local development. By making it easier to use the `Justfile` will be a positive step in that direction. I would suggest lets add the doc required for this PR in `$ just --list` and then take up the other things required to unify things.

Reply:
Okay. So you're suggesting to simply add docs to the recipes in this PR?